robot/rover/RoverSetup.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+# Script which setups the Space Concordia Robotics Software team's OBC.
+
+APPEND_TO_BASH="
+
+export ROS_MASTER_URI=http://localhost:11311
+export ROS_HOSTNAME=$USER
+
+. ~/Programming/robotics-prototype/robot/rospackages/devel/setup.bash
+. ~/Programming/robotics-prototype/venv/bin/activate
+source ~/Programming/robotics-prototype/robot/basestation/config/.bash_aliases
+
+"
+
+FINAL_MESSAGE="The script will now exit, ensure that ros is install correctly and that
+all of the rospackages are present. Reboot to run systemd services."
+
+REPO="/home/$USER/Programming/robotics-prototype"
+
+## START
+
+#check if in proper directory
+SCRIPT_DIRECTORY=$REPO/EnvironmentSetup.sh
+if [ ! -f $SCRIPT_DIRECTORY ]
+then
+   # -e enables text editing, \e[#m sets a text colour or background colour. \e[0 ends the edit.
+   echo -e "\e[31m\e[47mYou did not setup the repo directory correctly. Refer to README\e[0m"
+   exit 1
+fi
+
+
+#install prereqs
+sudo add-apt-repository ppa:deadsnakes/ppa -y
+sudo apt update -y
+sudo apt install python3.6 python3.6-venv nodejs npm openssh-server -y
+sudo ufw allow ssh
+
+# Setup venv
+cd $REPO
+python3.6 -m venv venv
+source venv/bin/activate
+
+# Install Requirements
+pip install -U pip
+pip install pyserial
```

Comment:
is there a reason why pip is installed here and not ran through pip -r requirements? 

Also you don't include a version # which could be potentially bad

Reply:
Not all of the python libraries are required, so there's no point to create a requirements.txt file just for one library. I included the version # thanks